src/test/java/com/gildedrose/GildedRoseTest.java

```diff
@@ -2,16 +2,160 @@
 
 import static org.junit.Assert.*;
 
+import org.junit.Before;
 import org.junit.Test;
 
 public class GildedRoseTest {
 
+    private static final String ITEM_DEFAULT_NAME = "foo";
+    private static final int ITEM_DEFAULT_SELLIN = 15;
+    private static final int ITEM_DEFAULT_QUALITY = 15;
+    private static final int ITEM_OVERDUE_SELLIN = 0;
+    private static final int ITEM_LONG_OVERDUE_SELLIN = -50;
+    private static final int ITEM_QUALITY_MAX = 50;
+    private static final String ITEM_NAME_BACKSTAGE_PASS = "Backstage passes to a TAFKAL80ETC concert";
+    private static final String ITEM_NAME_AGED_BRIE = "Aged Brie";
+    private static final String ITEM_NAME_SULFURAS = "Sulfuras, Hand of Ragnaros";
+    private static final int ITEM_QUALITY_MIN = 0;
+    private Item item;
+    private GildedRose app;
+
+    @Before
+    public void setUp() {
```

Comment:
We usually avoid global test setup methods and apply it in each test directly, making use of meaningful test utilities.